components/lib/biometric-prompt/src/main/java/mozilla/components/lib/auth/BiometricUtils.kt

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.lib.auth
+
+import android.content.Context
+import android.os.Build
+import androidx.annotation.VisibleForTesting
+import androidx.biometric.BiometricManager
+
+class BiometricUtils {
+
+    @VisibleForTesting
+    internal fun getAndroidBiometricManager(context: Context): BiometricManager {
+        return BiometricManager.from(context)
+    }
+
+    /**
+     * Checks if the appropriate SDK version and hardware capabilities are met to use the feature.
+     */
+    fun canUseFeature(context: Context): Boolean {
```

Comment:
Since we only want this method, can we just make this one an extension function so that we don't have to expose a utils class in the component for only this one?

We try to avoid util classes because they have potential to be a dumping ground for functions that never end up getting used anywhere else. With extension functions, they are more likely to be used (in my biased experience) since the autocomplete for the IDE lists them out as an option.

Reply:
If I do this I can't mock with Mockito extension functions . I need to use mockk . @jonalmeida

Reply:
Replied back to this thread in another one here: https://github.com/mozilla-mobile/android-components/pull/12291#discussion_r900160623